python-sdk/docs/astro/files/operators/get_file_list.rst

```diff
@@ -1,8 +1,8 @@
 .. _get_file_list:
 
-=============
-get_file_list
-=============
+=========================================================================
+:py:func:`get_file_list <astro.files.get_file_list>`
+=========================================================================
```

Comment:
Can we remove the extra `=======` spanning more than the text content?

Reply:
removed